### PR TITLE
New version: Microsoft.dotnetpreview 7.0-preview 3

### PR DESCRIPTION
**Pull request by lychichem**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/57973)

### CODE REVIEW

**Comment by lychichem** [comment]
cc @Ilia-Kosenkov in #57940

File: manifests/m/Microsoft/dotnetPreview/7.0.100.22179/Microsoft.dotnetPreview.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: exe
+#  Scope: 
+  InstallerUrl: https://download.visualstudio.microsoft.com/download/pr/6005cba0-c667-4412-a2e0-d2ca888a7733/712513654db361758e5f724258dcca19/dotnet-sdk-7.0.100-preview.3.22179.4-win-x64.exe
+  InstallerSha256: BC4C991AC9297F615300F69632DBA02D514745A62120FB860CAA5E2EA7739E00
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/dotnetPreview/7.0.100.22179/Microsoft.dotnetPreview.installer.yaml:19]:
```suggestion
  InstallerSha256: 060A8EFB14D48E0A0E38E9E17A9A50811458D11ABA9494AF639BF90705035186
```

File: manifests/m/Microsoft/dotnetPreview/7.0.100.22179/Microsoft.dotnetPreview.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Created using YamlCreate.ps1 v1.1.7
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.dotnetPreview
+PackageVersion: 7.0.100.22179
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/dotnetPreview/7.0.100.22179/Microsoft.dotnetPreview.installer.yaml:5]:
```suggestion
PackageVersion: 7.1.22.17904
```

File: manifests/m/Microsoft/dotnetPreview/7.0.100.22179/Microsoft.dotnetPreview.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Created using YamlCreate.ps1 v1.1.7
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.dotnetPreview
+PackageVersion: 7.0.100.22179
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/dotnetPreview/7.0.100.22179/Microsoft.dotnetPreview.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 7.1.22.17904
```

File: manifests/m/Microsoft/dotnetPreview/7.0.100.22179/Microsoft.dotnetPreview.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Created using YamlCreate.ps1 v1.1.7
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Microsoft.dotnetPreview
+PackageVersion: 7.0.100.22179
```
Comment (ItzLevvie) [inline manifests/m/Microsoft/dotnetPreview/7.0.100.22179/Microsoft.dotnetPreview.yaml:5]:
```suggestion
PackageVersion: 7.1.22.17904
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by Ilia-Kosenkov** [comment]
@lychichem , sorry I didn't respond in time, it seems we have a time zone difference. Did you get all the information you need?

**Review by ItzLevvie — APPROVED** [review]

**Review by raghavthind2005 — APPROVED** [review]

**Comment by lychichem** [comment]
> @lychichem , sorry I didn't respond in time, it seems we have a time zone difference. Did you get all the information you need?

Yes! you can now check updates by typing `winget upgrade`

**Comment by Ilia-Kosenkov** [comment]
@lychichem, just tested -- works like a charm! Thank you

**Pull request merged**